frontend/packages/kubevirt-plugin/src/statuses/vm/vm-status.ts

```diff
@@ -143,15 +142,16 @@ const isBeingMigrated = (
 const isBeingImported = (
   vm: VMKind,
   pods?: PodKind[],
+  pvcs?: PersistentVolumeClaimKind[],
   dataVolumes?: V1alpha1DataVolume[],
 ): VMStatusBundle => {
-  const importerPods = getVMImporterPods(vm, pods);
+  const importerPods = getVMImporterPods(vm, pods, pvcs);
   if (importerPods && importerPods.length > 0 && !isVMCreated(vm)) {
     const dvLookup = createBasicLookup(dataVolumes, getName);
     const importerPodsStatuses = importerPods.map((pod) => {
       const podStatus = getPodStatus(pod);
-      const dvName = getLabelValue(pod, `${CDI_KUBEVIRT_IO}/${STORAGE_IMPORT_PVC_NAME}`);
-      const dataVolume = dvLookup[dvName];
+      const pvcName = getPvcByImporterPodName(pvcs, getName(pod));
```

Comment:
this logic here is getting too tangled. Lets return a map from getVMImporterPods `{ [pvcName]: pod }`

Reply:
Done!